upup/models/cloudup/resources/addons/node-termination-handler.aws/k8s-1.11.yaml.template

```diff
@@ -291,103 +298,118 @@ spec:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: ENABLE_PROBES_SERVER
+              value: "false"
+            - name: PROBES_SERVER_PORT
+              value: "8080"
+            - name: PROBES_SERVER_ENDPOINT
+              value: "/healthz"
+            - name: LOG_LEVEL
+              value: "info"
+            - name: JSON_LOGGING
+              value: "true"
+            - name: LOG_FORMAT_VERSION
+              value: "1"
+            - name: ENABLE_PROMETHEUS_SERVER
+              value: "{{ WithDefaultBool .EnablePrometheusMetrics false }}"
+            - name: PROMETHEUS_SERVER_PORT
+              value: "9092"
+            - name: METADATA_TRIES
+              value: "3"
+            - name: DRY_RUN
+              value: "false"
+            - name: CORDON_ONLY
+              value: "false"
+            - name: TAINT_NODE
+              value: "false"
+            - name: EXCLUDE_FROM_LOAD_BALANCERS
+              value: "{{ WithDefaultBool .ExcludeFromLoadBalancers true }}"
+            - name: DELETE_LOCAL_DATA
+              value: "true"
+            - name: IGNORE_DAEMON_SETS
+              value: "true"
+            - name: POD_TERMINATION_GRACE_PERIOD
+              value: "-1"
+            - name: NODE_TERMINATION_GRACE_PERIOD
+              value: "120"
+            - name: EMIT_KUBERNETES_EVENTS
+              value: "true"
+            - name: ENABLE_SPOT_INTERRUPTION_DRAINING
+              value: "{{ WithDefaultBool .EnableSpotInterruptionDraining true }}"
+            - name: ENABLE_SCHEDULED_EVENT_DRAINING
+              value: "{{ WithDefaultBool .EnableScheduledEventDraining false }}"
+            - name: ENABLE_REBALANCE_MONITORING
+              value: "{{ WithDefaultBool .EnableRebalanceMonitoring false }}"
+            - name: ENABLE_REBALANCE_DRAINING
+              value: "{{ WithDefaultBool .EnableRebalanceDraining false }}"
+            - name: ENABLE_SQS_TERMINATION_DRAINING
+              value: "false"
+            - name: UPTIME_FROM_FILE
+              value: "/proc/uptime"
+          resources:
+            requests:
+              cpu: {{ .CPURequest }}
+              memory: {{ .MemoryRequest }}
+          volumeMounts:
+            - name: uptime
+              mountPath: /proc/uptime
+              readOnly: true
+      volumes:
+        - name: uptime
+          hostPath:
+            path: /proc/uptime
       nodeSelector:
         kubernetes.io/os: linux
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: eks.amazonaws.com/compute-type
+                operator: NotIn
+                values:
+                - fargate
```

Comment:
Do we really need/want this?

Reply:
It was in the upstream manifest and didn't seem harmful, but whatever.

Reply:
Ok, let's keep it to make it easier to diff with future updates.